pkg/config/cluster.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import (
+	"reflect"
+
+	"github.com/ethersphere/beekeeper/pkg/bee"
+)
+
+// Cluster represents cluster configuration
+type Cluster struct {
+	// parrent to inherit settings from
```

Comment:
parent. also, is the linter configured in this repo with misspell?

Reply:
fixed